src/lektorium/client/src/components/ControlPanel.vue

```diff
@@ -29,7 +29,8 @@
               <td>
                 <a
                   v-if="site.productionUrl.startsWith('http')"
-                  :href="site.productionUrl"
+                  href="#"
+                  @click="openNewTab(site.productionUrl)"
```

Comment:
Why not?
```
target="_blank"
```

Reply:
I thought it was unsafe to use "_blank"

Reply:
Any links to describe safety problem?

Reply:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
> https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Have you read the article?

Reply:
Yes, it looks like we don`t need it in our case. Reinsured, will fix it. Thank you!